tpp-app/src/services/CalendarService.js

```diff
@@ -0,0 +1,120 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { KEYS, TRACK_SYMPTOMS } from './utils/constants.js'
+import { Symptoms } from './utils/models.js';
+
+/**
+ * Makes API request to AsyncStorage to access symptoms for the entire year
+ * @param year: number
```

Comment:
Take a look at this [link](https://jsdoc.app/tags-param.html). Basically u can do type notation on an argument like `@param {number} year DESCRIPTION`. Makes it show up when u mouse over the signature in VS code & prob other editors. very cool, very hip

Reply:
ur old